scripts/umbrel-os/status-server/static/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Umbrel</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width,initial-scale=1.0">
+  <meta name="robots" content="noindex, nofollow" />
+  <meta name="referrer" content="no-referrer" />
+  <link rel="icon" href="/favicon.png">
+  <link rel="stylesheet" href="styles.css">
+  <meta name="description" content="Welcome back">
+</head>
+
+<body>
+  <noscript>
+    <strong>We're sorry but Umbrel
```

Comment:
```suggestion
    <strong>We're sorry, but Umbrel
```

Isn't that required here?

Reply:
It's just pulled straight from https://github.com/getumbrel/umbrel-dashboard/blob/c36af7cb58bec8d0533178675886a63f4a1fcc88/public/index.html